examples/complete/components/terraform/infra/vpc-flow-logs-bucket/component.yaml

```diff
@@ -0,0 +1,60 @@
+#  Including the file 'main.tf' since it matches the '**/*.tf' pattern from 'included_paths'
+#  Including the file 'outputs.tf' since it matches the '**/*.tf' pattern from 'included_paths'
+#  Including the file 'providers.tf' since it matches the '**/*.tf' pattern from 'included_paths'
+#  Including the file 'variables.tf' since it matches the '**/*.tf' pattern from 'included_paths'
+#  Including the file 'versions.tf' since it matches the '**/*.tf' pattern from 'included_paths'
+#
+# Pulling the mixing 'https://raw.githubusercontent.com/cloudposse/terraform-null-label/0.25.0/exports/context.tf'
+#  for the component 'infra/vpc-flow-logs-bucket' and writing to 'examples/complete/components/terraform/infra/vpc-flow-logs-bucket'
+# Pulling the mixing 'https://raw.githubusercontent.com/cloudposse/terraform-aws-components/0.194.0/modules/datadog-agent/introspection.mixin.tf'
+#  for the component 'infra/vpc-flow-logs-bucket' and writing to 'examples/complete/components/terraform/infra/vpc-flow-logs-bucket'
+
+apiVersion: atmos/v1
+kind: ComponentVendorConfig
+metadata:
+  name: vpc-flow-logs-bucket-vendor-config
+  description: Source and mixins config for vendoring of 'vpc-flow-logs-bucket' component
+spec:
+  source:
+    # 'uri' supports all protocols (local files, Git, Mercurial, HTTP, HTTPS, Amazon S3, Google GCP),
+    # and all URL and archive formats as described in https://github.com/hashicorp/go-getter
+    # In 'uri', Golang templates are supported  https://pkg.go.dev/text/template
+    # If 'version' is provided, '{{.Version}}' will be replaced with the 'version' value before pulling the files from 'uri'
+    uri: github.com/cloudposse/terraform-aws-components.git//modules/vpc-flow-logs-bucket?ref={{.Version}}
+    version: 0.194.0
+    # Only include the files that match the 'included_paths' patterns
+    # If 'included_paths' is not specified, all files will be matched except those that match the patterns from 'excluded_paths'
+    # 'included_paths' support POSIX-style Globs for file names/paths (double-star `**` is supported)
+    # https://en.wikipedia.org/wiki/Glob_(programming)
+    # https://github.com/bmatcuk/doublestar#patterns
+    included_paths:
+      - "**/*.tf"
+      - "**/*.tfvars"
+      - "**/*.md"
+    # Exclude the files that match any of the 'excluded_paths' patterns
+    # Note that we are excluding 'context.tf' since a newer version of it will be downloaded using 'mixins'
+    # 'excluded_paths' support POSIX-style Globs for file names/paths (double-star `**` is supported)
+    excluded_paths:
+      - "**/context.tf"
+
+  # mixins override files from 'source' with the same 'filename' (e.g. 'context.tf' will override 'context.tf' from the 'source')
+  # mixins are processed in the order they are declared in the list
+  mixins:
+    # https://github.com/hashicorp/go-getter/issues/98
+    - uri: https://raw.githubusercontent.com/cloudposse/terraform-null-label/0.25.0/exports/context.tf
```

Comment:
are the contents of files templates?

Reply:
not the whole file for now, just `uri`, we can improve it later